packages/treat/theme.d.ts

```diff
@@ -1,3 +1,7 @@
+type AnyIfEmpty<T extends object> = keyof T extends never ? any : T;
+
 declare module 'treat/theme' {
-  export type Theme = any;
+  export interface Theme {}
+  export type ThemeOrAny = AnyIfEmpty<Theme>
```

Comment:
is there another place you'd prefer to put this?

Reply:
As long as it works for all scenarios I'm comfortable with it here. Just testing that this works across the board atm.